Test/Payouts/PayoutsGetTest.cs

```diff
@@ -33,5 +34,34 @@ public async void TestPayoutsGetRequest()
 
             // Add your own checks here
         }
+
+        [Fact]
+        public async void TestPayoutsGetFailureRequest()
+        {
+
+            PayoutsGetRequest request = new PayoutsGetRequest("123")
+                .Page(4)
+                .PageSize(2)
+                .TotalRequired(true);
+
+            
+            try {
+                HttpResponse getResponse = await TestHarness.client().Execute(request);
+            } catch(Exception ex){
```

Comment:
may fail when there is a non HttpException raised

Reply:
updated